### ISSUE TITLE
Output shape 1x84x8400 does not match the expected shape of 1x5x8400.This model will not work with Aimmy, please use an ONNX V8 model.

### ISSUE THREAD

**Issue by xiaoxiongweihu** [Babyhamsta/Aimmy#1328]
Output shape 1x84x8400 does not match the expected shape of 1x5x8400.This model will not work with Aimmy, please use an ONNX V8 model.

**Comment by theratstyle** [comment]
Same issue

**Comment by ERROR200013** [comment]
Go to the next path: C:\Users\username\Desktop\a folder for training\runs\detect\nametrain\weights
And open in current folder cmd/terminal/PowerShell write the command: yolo export model=best.pt format=onnx

**Comment by theratstyle** [comment]
> Go to the next path: C:\Users\username\Desktop\a folder for training\runs\detect\nametrain\weights And open in current folder cmd/terminal/PowerShell write the command: yolo export model=best.pt format=onnx ( in folder has two files: best.pt and last.pt, you need best.tp to export file )

`PS C:\Users\qrtl\runs\detect\data5\weights> yolo export model=best.pt format=onnx
Ultralytics YOLOv8.1.5 🚀 Python-3.10.11 torch-2.1.2+cu121 CPU (AMD Ryzen 5 5600 6-Core Processor)
Model summary (fused): 168 layers, 3006038 parameters, 0 gradients, 8.1 GFLOPs

PyTorch: starting from 'best.pt' with input shape (1, 3, 640, 640) BCHW and output shape(s) **(1, 6, 8400)** (6.0 MB)

ONNX: starting export with onnx 1.15.0 opset 17...
ONNX: export success ✅ 0.6s, saved as 'best.onnx' (11.7 MB)

Export complete (2.1s)
Results saved to C:\Users\qrtl\runs\detect\data5\weights
Predict:         yolo predict task=detect model=best.onnx imgsz=640
Validate:        yolo val task=detect model=best.onnx imgsz=640 data=C:\Users\qrtl\Desktop\stalc\data.yaml
Visualize:       https://netron.app
💡 Learn more at https://docs.ultralytics.com/modes/export`

Nothing changed, we already have done this. We need reshape our onnx model to 1 5 8400.

**Comment by Babyhamsta** [comment]
Are you training a YOLOv8 model? All YOLOv8 model conversions should be outputting to the Onnx shape of 1x5x8400.

**Comment by theratstyle** [comment]
I don't now why, but used another YOLOv8 dataset to train model and everything fine.

Working model with shape 1x5x8400
` 
yolo export model=best.pt format=onnx
Ultralytics YOLOv8.1.5 🚀 Python-3.10.11 torch-2.1.2+cu121 CPU (AMD Ryzen 5 5600 6-Core Processor)
Model summary (fused): 168 layers, 3005843 parameters, 0 gradients, 8.1 GFLOPs

PyTorch: starting from 'best.pt' with input shape (1, 3, 640, 640) BCHW and output shape(s) (1, 5, 8400) (6.0 MB)

ONNX: starting export with onnx 1.15.0 opset 17...
ONNX: export success ✅ 0.6s, saved as 'best.onnx' (11.7 MB)

Export complete (2.2s)
Results saved to C:\Users\qrtl\Desktop\sc\runs\detect\data\weights
Predict:         yolo predict task=detect model=best.onnx imgsz=640
Validate:        yolo val task=detect model=best.onnx imgsz=640 data=C:\Users\qrtl\Desktop\sc\data.yaml
 `

Not working model  with shape 1x6x8400
`
yolo export model=best.pt format=onnx
Ultralytics YOLOv8.1.5 🚀 Python-3.10.11 torch-2.1.2+cu121 CPU (AMD Ryzen 5 5600 6-Core Processor)
Model summary (fused): 168 layers, 3006038 parameters, 0 gradients, 8.1 GFLOPs

PyTorch: starting from 'best.pt' with input shape (1, 3, 640, 640) BCHW and output shape(s) (1, 6, 8400) (6.0 MB)

ONNX: starting export with onnx 1.15.0 opset 17...
ONNX: export success ✅ 0.6s, saved as 'best.onnx' (11.7 MB)

Export complete (2.1s)
Results saved to C:\Users\qrtl\Desktop\stalc2\runs\detect\data\weights
Predict:         yolo predict task=detect model=best.onnx imgsz=640
Validate:        yolo val task=detect model=best.onnx imgsz=640 data=C:\Users\qrtl\Desktop\stalc2\data.yaml
`

**Comment by Babyhamsta** [comment]
Did they update the shape? I'll do some testing with the latest updates.
